From: Modestas Vainius <modestas@vainius.eu>
Subject: Fix bashisms in CMake scripts.
 Fix bashisms found in /bin/sh scripts. Thanks to Raphael Geissert for heads
 up.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=530056

--- a/Modules/CPack.RuntimeScript.in
+++ b/Modules/CPack.RuntimeScript.in
@@ -3,10 +3,10 @@
 # Modified from: Aaron Voisine <aaron@voisine.org>
 
 CWD="`dirname \"$0\"`"
-TMP=/tmp/$UID/TemporaryItems
+TMP=/tmp/$(id -ru)/TemporaryItems
 
 version=`sw_vers -productVersion`
-if [ "$?" == "0" ]; then
+if [ "$?" = "0" ]; then
   major=${version%%\.*}
   rest=${version#*\.}
   minor=${rest%%\.*}
@@ -24,7 +24,7 @@
 
 
 # if 10.5 or greater, then all the open-x11 stuff need not occur
-if ((( $major < 10 )) || ((( $major == 10)) && (( $minor < 5 )))); then
+if [ "$major" -lt 10 ] || ([ "$major" -eq 10 ] && [ "$minor" -lt 5 ]); then
 version=`sw_vers -productVersion`
 if [ "$?" = "0" ]; then
   major=${version%%\.*}
@@ -44,7 +44,7 @@
 
 
 # if 10.5 or greater, then all the open-x11 stuff need not occur
-if ((( $major < 10 )) || ((( $major == 10)) && (( $minor < 5 )))); then
+if [ "$major" -lt 10 ] || ([ "$major" -eq 10 ] && [ "$minor" -lt 5 ]); then
 ps -wx -ocommand | grep -e '[X]11.app' > /dev/null
 if [ "$?" != "0" -a ! -f ~/.xinitrc ]; then
     echo "rm -f ~/.xinitrc" > ~/.xinitrc
